pkg/api/meta/help_test.go

```diff
@@ -114,7 +115,10 @@ type fakePtrInterfaceList struct {
 	Items *[]runtime.Object
 }
 
-func (f fakePtrInterfaceList) IsAnAPIObject() {}
+func (obj fakePtrInterfaceList) SetGroupVersionKind(gvk *unversioned.GroupVersionKind) {}
```

Comment:
There's been at least one complaint about `gvk` and defending the choice isn't easy to do.

@derekwaynecarr wrote a treatise on names https://github.com/kubernetes/kubernetes/pull/18379#discussion_r47100949 and on reflection, I do like `kind` better.